src/components/include/cos_kernel_api.h

```diff
@@ -51,6 +51,7 @@ void cos_compinfo_init(struct cos_compinfo *ci, pgtblcap_t pgtbl_cap, captblcap_
  */
 void cos_meminfo_init(struct cos_meminfo *mi, vaddr_t untyped_ptr, unsigned long untyped_sz, pgtblcap_t pgtbl_cap);
 void cos_meminfo_alloc(struct cos_compinfo *ci, vaddr_t untyped_ptr, unsigned long untyped_sz);
+vaddr_t __bump_mem_expand_range(struct cos_compinfo *ci, pgtblcap_t cipgtbl, vaddr_t mem_ptr, unsigned long mem_sz);
```

Comment:
Given my previous comments, we might be able to get rid of you needing to do this.

Otherwise, @phanikishoreg  is right.  We should make a public function, likely called `cos_pgtbl_intern_alloc(...)` that will simply call the private `__` function.  This is just a namespace/consistency issue, not a technical one.